mix.exs

```diff
@@ -47,6 +47,25 @@ defmodule Broadway.MixProject do
       groups_for_extras: [
         Examples: Path.wildcard("guides/examples/*.md"),
         Internals: Path.wildcard("guides/internals/*.md")
+      ],
+      groups_for_modules: [
+        # Ungrouped Modules:
+        #
+        # Broadway
+        # Broadway.Message
+
+        Acknowledgement: [
+          Broadway.Acknowledger,
+          Broadway.CallerAcknowledger,
+          Broadway.NoopAcknowledger
+        ],
+        Batching: [
+          Broadway.BatchInfo
```

Comment:
Let's keep this at the root, as it is part of the "main" API?

Reply:
done.